tensorflow/core/framework/function_testlib.cc

```diff
@@ -91,6 +91,23 @@ FunctionDef IsZero() {
       });
 }
 
+FunctionDef IsZeroV2() {
```

Comment:
Out of curiosity, why do we need a new one instead of reusing the existing one?

Reply:
@jsimsa The difference is that the existing one `IsZero` defines the input as `{"x: T"}`; `IsZeroV2` defines the input as `{"x: int64"}`. 

If we use `IsZero` as the first function, there will be some issue in [fusion_utils::LazyConjunctionNodes()](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/grappler/optimizers/data/fusion_utils.cc#L387-L388): `input_arg.type()` will return DT_INVALID and then the attr `Tin` of the cond node (which is `If` op) will be `DT_INVALID`, which will break the conversion of the cond node to lower level ops (e.g. Switch, Merge).

It looks like [fusion_utils::FuseFunctions()](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/grappler/optimizers/data/fusion_utils.cc#L431-L432) wants to stop the fusion for the cases similar with `IsZero()` but `first_function.attr_size()` always return 0 (I'm not sure my understanding is correct here).